airgun/widgets.py

```diff
@@ -1210,7 +1210,9 @@ def read(self):
 
 
 class SatTable(Table):
-    """Satellite version of table. Main difference - in case it's empty, there
+    """DEPRECATED. Please use :class:`widgetastic.widget.Table` instead.
```

Comment:
No longer needed as now colspans/rowspan are correctly handled on widgetastic side, reading such tables does not cause exceptions due to improper rows/columns count.
Moreover, our implementation causes issues with fillable dynamic tables (like content view filter rules one), as adding extra row doesn't remove `no_rows_message` row immediately, and its presence makes it impossible to read such table:
![image](https://user-images.githubusercontent.com/11719030/49581524-e0e0c580-f95a-11e8-8657-6d8a9d95d718.png)